### ISSUE TITLE
Servo Output Functions Page (w/ Control?)

### ISSUE THREAD

**Issue by manavgandhi17** [mavlink/qgroundcontrol#10199]
Would be nice to have a page where you can set the servo outputs to different functions. That way when I am setting up my planes in ArduPlane I don’t have to mess with the parameter list for basic setup. Don’t know if this should be its own page - or merged into the Motors page and renamed to “Actuators” or something like that.

**Comment by AhmedElsafy** [comment]
Have you tried flashing the version 1.13 beta developer, you will find a page in QGC with actuator allocation that you can assign for each group of actuators the mode they work in

**Comment by manavgandhi17** [comment]
I have tested on both the current stable and latest daily build. What do you mean v1.13? I am running ArduPlane.

**Comment by AhmedElsafy** [comment]
Sorry my bad, I was more referring to PX4, it has that feature to control every output.

**Comment by AhmWael** [comment]
I’m interested in implementing a Servo Outputs setup page for ArduPilot similar to Mission Planner. Is there any design preference or existing work I should be aware of before starting? @HTRamsey

**Comment by DonLakeFlyer** [comment]
Just look at the existing Vehicle Configuration Qml pages and follow them for code/ui consistency.
